generate_tuxsuite.py

```diff
@@ -59,11 +54,11 @@ def emit_tuxsuite_yml(config, tree):
     distribution_configs = []
     allconfigs = []
     for build in config["builds"]:
-        if build["git_repo"] == repo and build["git_ref"] == ref:
+        if build["git_repo"] == repo and build["git_ref"] == ref and build[
```

Comment:
the wrapping here is a bit jarring. Can the line end with `and` so that `build["llvm_version"]` starts the next line?

Reply:
This is just `yapf` does.

I could do this if you would prefer?

```diff
diff --git a/generate_tuxsuite.py b/generate_tuxsuite.py
index 7b5ecab..da1b43c 100755
--- a/generate_tuxsuite.py
+++ b/generate_tuxsuite.py
@@ -54,8 +54,9 @@ def emit_tuxsuite_yml(config, tree, llvm_version):
     distribution_configs = []
     allconfigs = []
     for build in config["builds"]:
-        if build["git_repo"] == repo and build["git_ref"] == ref and build[
-                "llvm_version"] == llvm_version:
+        if build["git_repo"] == repo and \
+           build["git_ref"] == ref and \
+           build["llvm_version"] == llvm_version:
             arch = build["ARCH"] if "ARCH" in build else "x86_64"
             if llvm_version == max_version:
                 toolchain = "clang-nightly"
```

Reply:
that looks nicer; won't `yapf` undo that though without annotations?

Reply:
Does not appear so. I've included it for the next push.